example/envoy-config.yaml

```diff
@@ -3,23 +3,26 @@ static_resources:
     - address:
         socket_address:
           address: 0.0.0.0
-          port_value: 8001
+          port_value: 80
```

Comment:
port 80 might not be the best choice for an example. One usually finds it occupied.

Reply:
I was mainly taking into account the usage of the example via docker-compose. In that case we will use ports `8080` and `8081` respectively for envoy and httpbin. Is it better to take into account also a standalone usage of the envoy config file?